src/pybind/mgr/mgr_util.py

```diff
@@ -23,6 +27,283 @@
 logger = logging.getLogger(__name__)
 
 
+class CephfsConnectionException(Exception):
+    def __init__(self, error_code, error_message):
+        self.errno = error_code
+        self.error_str = error_message
+
+    def to_tuple(self):
+        return self.errno, "", self.error_str
+
+    def __str__(self):
+        return "{0} ({1})".format(self.errno, self.error_str)
+
+
+class ConnectionPool(object):
```

Comment:
Adding a class `ConnectionPool` in mgr_util.py doesn't imply that this is supposed to be CephFS specific. BTW it is posible to import the python code from other modules easily. Why not leave the code where it is today and just import it from other mgr modules?

Reply:
I think we should stick to keeping this class in mgr_utils and abstract it in `CephFSUtils` or something.